src/com_tjvendors/site/views/vendor/tmpl/default.php

```diff
@@ -18,7 +18,7 @@
 			?>
 	</h1>
 	<input type="hidden" name="client" value="<?php echo $this->input->get('client', '', 'STRING'); ?>" />
-	<div class="profile row" id="tjv-wrapper">
+	<div class="profile row tjBs3" id="tjv-wrapper">
```

Comment:
declare a constant for classname in entry file, use constant in views

Reply:
Updated everywhere